packages/block-library/src/details/edit.js

```diff
@@ -90,11 +90,13 @@ function DetailsEdit( { attributes, setAttributes, clientId } ) {
 					) }
 				/>
 			</InspectorControls>
-			<details
-				{ ...innerBlocksProps }
-				open={ hasSelection || showContent }
-			>
-				<summary onClick={ ( event ) => event.preventDefault() }>
+			<details { ...innerBlocksProps } open={ isOpen }>
+				<summary
+					onClick={ ( event ) => {
+						event.preventDefault();
+						setIsOpen( ! isOpen );
```

Comment:
Won't this change here have the sideeffect that clicking into the Summary Rich Text element is going to open / close the details element?

Reply:
This shouldn't cause any issues, since I haven't added the `name` attribute to the block in the editor.

However, it's worth discussing whether we should add the `name` attribute behavior in the editor as well, to be consistent with the frontend.

Reply:
Ignore me, I seem to have commented on a disconnected HEAD.